tests/end2end/data/fileselect/fileselect.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```

Comment:
Since we now have a single `fileselect.html`, I suppose it doesn't make much sense to have it in a subfolder.

Reply:
Moved file and one level up